content/docs/sidebar.json

```diff
@@ -138,7 +138,7 @@
       {
         "label": "Experiment Management",
         "slug": "experiment-management",
-        "source": "experiment-management/index.md",
+        "source": false,
```

Comment:
I dropped this page since it was pretty long and seemed like a lot of legacy info that distracted from getting to dvc exp features.

Reply:
I didn't see any particular content from it that needed to be kept anywhere, but we could also move pieces of it to other docs if needed.

Reply:
See my comment on moving the Overview to the index page instead. Thanks

Reply:
That's a much better idea, thanks!